### ISSUE TITLE
Why not implement batch normalization?

### ISSUE THREAD

**Issue by MarkYangjiayi** [shekkizh/FCN.tensorflow#107]
I noticed that in TensorflowUtils.py you wrote a batch_norm function but did not implement it. Does anybody know why? Is it because the original paper of FCN does not have batch_norm?
